dashboard/settings.py

```diff
@@ -115,19 +112,38 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
-BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I didn't change the value, just the name here in `settings.py` ... so the `.bowerrc` is still working.